aws/v17.1.0/release.yaml

```diff
@@ -0,0 +1,78 @@
+# Generated with:
+# devctl release create --name 17.1.0 --base 17.0.1 --provider aws --component cluster-operator@3.14.0 --component aws-operator@10.18.0 --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v17.1.0
+spec:
+  apps:
+  - componentVersion: 1.5.0
+    name: aws-ebs-csi-driver
+    version: 2.8.1
+  - name: cert-exporter
+    version: 2.0.1
+  - componentVersion: 1.6.1
+    name: cert-manager
+    version: 2.12.0
+  - name: chart-operator
+    version: 2.20.0
+  - name: cluster-autoscaler
+    version: 1.22.2-gs3
+  - componentVersion: 1.8.6
+    name: coredns
+    version: 1.8.0
+  - componentVersion: 0.10.2
+    name: external-dns
+    version: 2.9.0
+  - componentVersion: 4.2.0
+    name: kiam
+    version: 2.1.0
```

Comment:
What are we going to do about Kiam?

Reply:
good point, forgot about it, can we just bump the version? is that changed kiam working for clusters with and without IRSA?

Reply:
No, it's a variant that defaults the whitelist value. I didn't realise Kiam was part of the release, I'm now not sure if the variant is the correct approach.

I'm wondering if we leave Kiam as-is and include in the IRSA docs that the user-values need providing (like we did in the test)

Reply:
lets talk in planning